index.bs

```diff
@@ -2005,36 +2005,42 @@ enum IDBRequestReadyState {
         then returns {{"done"}}.
 </div>
 
-The <dfn attribute for=IDBRequest>result</dfn> attribute's getter must
-[=throw=] an "{{InvalidStateError}}" {{DOMException}} if [=/this=]'s [=request/done flag=] is
-false. Otherwise, the attribute's getter must return [=/this=]'s [=request/result=], or undefined if the
-request resulted in an error.
+<div class=algorithm>
+The <dfn attribute for=IDBRequest>result</dfn> getter steps are:
+
+1. If [=/this=]'s [=request/done flag=] is false, then [=throw=] an "{{InvalidStateError}}" {{DOMException}}.
+1. Otherwise, return [=/this=]'s [=request/result=], or undefined if the request resulted in an error.
 
+</div>
+
+<div class=algorithm>
+The <dfn attribute for=IDBRequest>error</dfn> getter steps are:
 
-The <dfn attribute for=IDBRequest>error</dfn> attribute's getter must
-[=throw=] an "{{InvalidStateError}}" {{DOMException}} if [=/this=]'s [=request/done flag=] is false.
-Otherwise, the attribute's getter must return [=/this=]'s [=request/error=],
-or null if no error occurred.
+1. If [=/this=]'s [=request/done flag=] is false, then [=throw=] an "{{InvalidStateError}}" {{DOMException}}.
+1. Otherwise, return [=/this=]'s [=request/error=], or null if no error occurred.
 
-The <dfn attribute for=IDBRequest>source</dfn> attribute's getter must
+</div>
+
+
+The <dfn attribute for=IDBRequest>source</dfn> getter steps are to
 return [=/this=]'s [=request/source=], or null if no
 [=request/source=] is set.
 
-The <dfn attribute for=IDBRequest>transaction</dfn> attribute's getter
-must return [=/this=]'s [=request/transaction=]. This
-property can be null for certain requests, such as for [=/requests=]
-returned from {{IDBFactory/open()}}.
+The <dfn attribute for=IDBRequest>transaction</dfn> getter steps are to
+return [=/this=]'s [=request/transaction=].
 
-The <dfn attribute for=IDBRequest>readyState</dfn> attribute's getter
-must return {{"pending"}} if [=/this=]'s [=request/done flag=] is false, and
+<aside class=note>
+    The {{IDBRequest/transaction}} property can be null for certain requests, such as for [=/requests=] returned from {{IDBFactory/open()}}.
```

Comment:
Maybe "The transaction getter can return null"? Both are correct though.

Reply:
done